sigma_fun/Cargo.toml

```diff
@@ -9,10 +9,11 @@ edition = "2018"
 [dependencies]
 generic-array = "0.14"
 digest = "0.9"
-secp256kfun = { path = "../secp256kfun", version = "^0.3.2-alpha.0", default-features = false, optional = true }
+secp256kfun = { path = "../secp256kfun", version = "^0.3.2-alpha.0", default-features = false, optional = true, features = ["proptest"] }
```

Comment:
You should be able to move this to `dev-dependencies` so that this feature is only enabled during testing.